src/diffusers_interpret/explainer.py

```diff
@@ -425,14 +480,75 @@ def _mimic_pipeline_call(
+
+        Raises:
+            NotImplementedError: This method must be implemented by the child class.
         """
         raise NotImplementedError
 
 
 class BasePipelineImg2ImgExplainer(BasePipelineExplainer):
     """
     Core base class to explain img2img and inpaint pipelines
+
+    Attributes:
+        model (`torch.nn.Module`):
+            The model to be explained.
+        tokenizer (`PreTrainedTokenizer`):
+            The tokenizer to be used.
+        text_encoder (`torch.nn.Module`):
+            The text encoder to be used.
+        image_encoder (`torch.nn.Module`):
+            The image encoder to be used.
+        image_decoder (`torch.nn.Module`):
+            The image decoder to be used.
+        safety_checker (`SafetyChecker`):
+            The safety checker to be used.
+        device (`torch.device`):
+            The device to be used.
+        model_type (`str`):
+            The model type to be used.
+        model_name (`str`):
+            The model name to be used.
+        model_version (`str`):
+            The model version to be used.
+        model_config (`DictConfig`):
+            The model config to be used.
+        model_config_path (`str`):
+            The model config path to be used.
+        model_checkpoint_path (`str`):
+            The model checkpoint path to be used.
+        model_checkpoint_version (`str`):
+            The model checkpoint version to be used.
+        model_checkpoint_name (`str`):
+            The model checkpoint name to be used.
+        model_checkpoint_url (`str`):
+            The model checkpoint url to be used.
+        model_checkpoint_cache_dir (`str`):
+            The model checkpoint cache dir to be used.
+        model_checkpoint_force_download (`bool`):
+            Whether or not to force download the model checkpoint.
+        model_checkpoint_force_extract (`bool`):
+            Whether or not to force extract the model checkpoint.
+        model_checkpoint_local_files_only (`bool`):
+            Whether or not to only use local files for the model checkpoint.
+        model_checkpoint_use_cdn (`bool`):
+            Whether or not to use the cdn for the model checkpoint.
+        model_checkpoint_kwargs (`DictConfig`):
+            The model checkpoint kwargs to be used.
+        model_checkpoint_map_location (`str`):
+            The model checkpoint map location to be used.
+        model_checkpoint_force_map_location (`bool`):
+            Whether or not to force map location for the model checkpoint.
+        model_checkpoint_ignore_mismatched_sizes (`bool`):
+            Whether or not to ignore mismatched sizes for the model checkpoint.
+        model_checkpoint_strict (`bool`):
+            Whether or not to use strict for the model checkpoint.
+        model_checkpoint_return_unused_kwargs (`bool`):
+            Whether or not to return unused kwargs for the model checkpoint.
+        model_checkpoint_kwargs

```

Comment:
These are actually the properties of `BasePipelineImg2ImgExplainer.pipe`, we can remove from here, right?